.travis.yml

```diff
@@ -0,0 +1,8 @@
+dist: trusty
```

Comment:
Does travis not support xenial at least? trusty seems... ancient.

Reply:
lmgtfy... no. :-D

Reply:
Trusty is very fresh w.r.t to Travis, https://blog.travis-ci.com/2017-12-12-new-trusty-images-q4-launch

Reply:
Wow, ok :)